taxi/README.md

```diff
@@ -0,0 +1,34 @@
+# Taxi Sample data
+
+This folder contains sample data files concerning taxi rides in NYC from October 31, 2018 and November 1, 2018 that work within the Deephaven console.
+
+## Table of contents
+
+- `taxi.parquet`: uncompressed parquet data format
+- `taxi.csv`: csv data format
+
+## Fields in the `taxi.parquet` file
+
+`station_data.csv` has 17 fields:
+
+  - **int64** VendorID
+  - **int64**  tpep_pickup_datetime (TIMESTAMP(MICROS,false))
+  - **int64**  tpep_dropoff_datetime (TIMESTAMP(MICROS,false))
+  - **int64**  passenger_count
+  - **double**  trip_distance
+  - **int64**  RatecodeID
+  - **binary**  store_and_fwd_flag (STRING)
+  - **int64**  PULocationID
+  - **int64**  DOLocationID
+  - **int64**  payment_type
+  - **double**  fare_amount
+  - **double**  extra
+  - **double**  mta_tax
+  - **double**  tip_amount
+  - **double**  tolls_amount
+  - **double**  improvement_surcharge
+  - **double**  total_amount
```

Comment:
Can we make the format consistent with what is here:  https://raw.githubusercontent.com/mikeblas/samples-junk/main/metriccentury/README.md

Something like:
```
- **<colname>(<type>):** <description>
```

or some other similar format that includes type.